src/components/Common/SearchInputBox/SearchInputBox.jsx

```diff
@@ -2,7 +2,7 @@ import React, { useRef, useState } from "react";
 import { SearchBox, SearchIcon, SearchInput, CancelIcon, CancelIconWrapper } from "./SearchInputBoxElements";
 import { FcSearch } from "react-icons/fc";
 
-const SearchInputBox = ({ placeholder, value, onChange, setValue }) => {
+const SearchInputBox = ({ placeholder, value, onChange, clearValue }) => {
```

Comment:
Every component that placed in Common folder, if you make a change you responsible to check where this component was used and change there accordingly.

https://github.com/thecyberworld/TheCyberHUB/assets/76536506/0636e80f-782d-486d-9353-61dc81ca2a7f

In future we plan to add storybook or similar for those components but still for now you should be aware that the components in this folder can be used in many places at the app.